runs/models.tsp

```diff
@@ -337,6 +479,14 @@ model RunObject {
   @encode("unixTimestamp", int32)
   completed_at: utcDateTime | null;
 
+  @doc("""
+  Details on why the run is incomplete. Will be `null` if the run is not incomplete.
+  """)
+  incomplete_details: {
```

Comment:
I believe the `status` enum property is missing `incomplete` in the swagger spec and we must add it manually. This is my interpretation for this new `incomplete_details` property to make sense and from looking at the run lifecycle here: https://platform.openai.com/docs/assistants/how-it-works/runs-and-run-steps

Reply:
Added a rule to do it manually. Very odd that it's missing like that, but I share your assessment.